jepsen/src/jepsen/tests/causal.clj

```diff
@@ -116,8 +116,7 @@
 (defn cw2 [_ _] {:type :invoke, :f :write, :value 2})
 
 (defn test [opts]
-  {:model (causal-register)
-   :checker (independent/checker (check))
+  {:checker (independent/checker (check (causal-register)))
```

Comment:
I need to redesign parts of the causal checker, it's kind of a mess RIP

Reply:
Is OK!